src/test/scala/org/hammerlab/guacamole/util/TestUtil.scala

```diff
@@ -231,11 +232,30 @@ object TestUtil extends Matchers {
       Pileup(normalReads, contig, locus, reference.getContig(contig)))
   }
 
-  def loadPileup(sc: SparkContext, filename: String, reference: ReferenceBroadcast, locus: Long = 0, contig: Option[String] = None): Pileup = {
-    val records = TestUtil.loadReads(sc, filename, reference = reference).mappedReads
+  def loadPileup(sc: SparkContext,
+                 filename: String,
+                 reference: ReferenceBroadcast,
+                 locus: Long = 0,
+                 contigOpt: Option[String] = None): Pileup = {
```

Comment:
I like the `Opt` suffix on `Option`s. Would you prefer I name this `contigOption` or just `contig` The former feels unnecessarily verbose to me, and the latter is awkward when you want to do anything with the Option:

```
contig.map(contig => …)
```

I'll make it `contigOption` if you prefer.